### ISSUE TITLE
Pinned version of aiohttp raises error on Python 3.5

### ISSUE THREAD

**Issue by sloria** [CenterForOpenScience/waterbutler#87]
When I ran `invoke server` in a Python 3.5 environment, I get the following error:

```
$ inv server                                                 
Traceback (most recent call last):
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/bin/inv", line 11, in <module>
    sys.exit(main())
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/lib/python3.5/site-packages/invoke/cli.py", line 438, in main
    dispatch(sys.argv)
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/lib/python3.5/site-packages/invoke/cli.py", line 431, in dispatch
    return executor.execute(*tasks)
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/lib/python3.5/site-packages/invoke/executor.py", line 100, in execute
    task=task, name=name, args=args, kwargs=kwargs, config=config
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/lib/python3.5/site-packages/invoke/executor.py", line 153, in _execute
    result = task(*args, **kwargs)
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/lib/python3.5/site-packages/invoke/tasks.py", line 111, in __call__
    result = self.body(*args, **kwargs)
  File "/Users/sloria/projects/cos-projects/waterbutler/tasks.py", line 60, in server
    from waterbutler.server.app import serve
  File "/Users/sloria/projects/cos-projects/waterbutler/waterbutler/server/app.py", line 9, in <module>
    from waterbutler.server.api import v0
  File "/Users/sloria/projects/cos-projects/waterbutler/waterbutler/server/api/v0/__init__.py", line 1, in <module>
    from waterbutler.server.api.v0 import zip
  File "/Users/sloria/projects/cos-projects/waterbutler/waterbutler/server/api/v0/zip.py", line 4, in <module>
    from waterbutler.server.api.v0 import core
  File "/Users/sloria/projects/cos-projects/waterbutler/waterbutler/server/api/v0/core.py", line 11, in <module>
    from waterbutler import tasks
  File "/Users/sloria/projects/cos-projects/waterbutler/waterbutler/tasks/__init__.py", line 2, in <module>
    from waterbutler.tasks.copy import copy
  File "/Users/sloria/projects/cos-projects/waterbutler/waterbutler/tasks/copy.py", line 4, in <module>
    from waterbutler.core import utils
  File "/Users/sloria/projects/cos-projects/waterbutler/waterbutler/core/utils.py", line 7, in <module>
    import aiohttp
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/lib/python3.5/site-packages/aiohttp/__init__.py", line 6, in <module>
    from . import hdrs  # noqa
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/lib/python3.5/site-packages/aiohttp/hdrs.py", line 2, in <module>
    from .multidict import upstr
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/lib/python3.5/site-packages/aiohttp/multidict.py", line 285, in <module>
    class _ItemsView(_ViewBase, abc.ItemsView):
  File "/Users/sloria/.local/share/virtualenvs/waterbutler/bin/../lib/python3.5/abc.py", line 133, in __new__
    cls = super().__new__(mcls, name, bases, namespace)
TypeError: multiple bases have instance lay-out conflict
```

Upgrading to the latest PyPI release (0.17.4) fixed the problem.

Glancing at the changelog (https://aiohttp.readthedocs.org/en/stable/changes.html), there don't appear to be breaking changes since 0.14.1 except that aiohttp dropped support for Python < 3.4.1.

**Comment by rdhyee** [comment]
I just ran into this problem when trying to use Python 3.5.

**Comment by felliott** [comment]
There's a branch to switch to python 3.5 (feature/python-3.5) that should be merged soon.  This has been resolved on that branch.

**Comment by whit537** [comment]
I believe this can be closed with #114, ya?

**Comment by felliott** [comment]
Yes it can. Thanks for the poke!
